### PR TITLE
initialize webapi project

### PR DESCRIPTION
**Pull request by vldmrklnk**
New WebApi project initialized. EntityFrameworkCore nuget packages added.

### CODE REVIEW

File: InternshipBE/InternshipBE/Program.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace InternshipBE
```
Comment (romanstat) [inline InternshipBE/InternshipBE/Program.cs]: project name is WebApi, but solution is InternshipBE

Comment (romanstat) [reply InternshipBE/InternshipBE/Program.cs]: solution
image

Comment (romanstat) [reply InternshipBE/InternshipBE/Program.cs]: inside project  WebApi
image

Comment (vldmrklnk) [reply InternshipBE/InternshipBE/Program.cs]: fixed this mistakes

**Review by romanstat — APPROVED** [review]
Resolve conflict with .gitignore and can merge

**Pull request merged**